spec/triggers/stan.md

```diff
@@ -0,0 +1,78 @@
+# How to test the Nats streaming scaler
```

Comment:
This is not really the best place for this, maybe move this to another location?

@ahmelsayed might be able to help you with this